src/main/resources/attackvectors/UnrestrictedFileUploadPayload.properties

```diff
@@ -0,0 +1,38 @@
+  <code>&lt;a:script xmlns:a="http://www.w3.org/1999/xhtml"&gt;alert(navigator.userAgent)&lt;/a:script&gt;</code><br/>\
+  Now if we upload this file and access it, the file should get executed in the browser.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_6=Hackers can execute code in a user's machine by uploading a file in the proper format to the server and trick the user into accessing it. Generally, users run personal machines with administrator privileges and disabled browser security policies which make attack execution easier. A persistent XSS attack is a similar type of attack that causes cookie and sensitive data thefts.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation is improved in this level where only files containing .jpeg and .png extension are accepted, so now to exploit this level you have to a create a file with extension .jpeg.html,and paste the following code: <br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(navigator.userAgent);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  Now if we upload this file and access it, the file should get executed in the browser.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_7=Hackers can execute code in a user's machine by uploading a file in the proper format to the server and trick the user into accessing it. Generally, users run personal machines with administrator privileges and disabled browser security policies which make attack execution easier. A persistent XSS attack is a similar type of attack that causes cookie and sensitive data thefts.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation is improved in this level where only files with .jpeg and .png extension are accepted, so now to exploit this level we have to create a file with extension like .html.jpeg, and insert a null byte character like '\0' or '%00' before .jpeg or .png and paste the following code: <br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(navigator.userAgent);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  The extension after the null byte is ignored and the file will be treated as .html. Now if we upload this file and access it, the file should get executed in the browser.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_8=A path traversal attack aims to overwrite files and directories that are stored outside the web root folder. By manipulating variables that reference files with "dot-dot-slash (../)" sequences and its variations or by using absolute file paths, it may be possible to change arbitrary files and directories stored on file system including application source code or configuration and critical system files.<br/>\
+<b>How to exploit this level?</b><br/>\
+  This level is vulnerable for path traversal, so if you choose a filename like <code>../index.html</code> then the file will be stored in another directory than provided.
```

Comment:
does this level allows HTML extension?

Reply:
Level 8? Yes

Reply:
oh, then it should be vulnerable to XSS right?

Reply:
I don't think. When you try to access it, the file is downloaded

Reply:
oh ok. 

thanks,
Karan